doc/api/assert.md

```diff
@@ -147,6 +147,67 @@ try {
 }
 ```
 
+## Class : assert.Tracker
```

Comment:
Just a nit: I would name this a bit more specifically so folks know what it's tracking. `CallTracker` perhaps?